tck/src/main/java/com/github/bsideup/liiklus/records/tests/ConsumerGroupTest.java

```diff
@@ -0,0 +1,71 @@
+package com.github.bsideup.liiklus.records.tests;
+
+import com.github.bsideup.liiklus.records.RecordStorageTestSupport;
+import com.github.bsideup.liiklus.records.RecordsStorage;
+import lombok.val;
+import org.junit.jupiter.api.Test;
+import reactor.core.Disposable;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.ReplayProcessor;
+
+import java.util.HashMap;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Function;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public interface ConsumerGroupTest extends RecordStorageTestSupport {
+
+    int getNumberOfPartitions();
+
+    @Test
+    default void testMultipleGroups() throws Exception {
+        val numberOfPartitions = getNumberOfPartitions();
+        assertThat(numberOfPartitions).as("number of partitions").isGreaterThanOrEqualTo(2);
```

Comment:
looks more like assumption, and not assertion. What if some record storages will want to have only 1 partition?

Reply:
good question. Ok, will make it an assumption 👍